src/auth-service/routes/v1/inquiries.js

```diff
@@ -69,8 +69,21 @@ router.post(
         .bail()
         .trim(),
       body("fullName")
-        .exists()
-        .withMessage("the fullName should be provided")
+        .optional()
```

Comment:
input validation for the part of creating inquiries, that is `GET INVOLVED `on the website.